aspnet/tutorials/nano-server.rst

```diff
@@ -1,8 +1,149 @@
+
+After setting the local password, you will manage Nano Server using the management tools remotely.
+
+Connecting to your Nano Server Instance using PowerShell Remote
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Open an elevated PowerShell window to add your remote Nano Server instance to your ``TrustedHosts`` list.
+
+.. code:: ps1
+
+  $ip = "10.83.181.14" # replace with the correct IP address
+  Set-Item WSMan:\localhost\Client\TrustedHosts "$ip" -Force
+
+Once you have added your Nano Server instance to your ``TrustedHosts``, you can connect to it using PowerShell Remote session
+
+.. code:: ps1
+
+  $ip = "10.83.181.14" # replace with the correct IP address
+  $s = New-PSSession -ComputerName $ip -Credential ~\Administrator
+  Enter-PSSession $s
+
+If you have successfully connected using PSRemote, you prompt will look like this ``[10.83.181.14]: PS C:\Users\Administrator\Documents>``
+
+Installing the HTTPPlatformHandler Module
+-----------------------------------------
+
+At the moment, the process to install the HTTPPlatformHandler Module for IIS is manual. You will need to install the latest 64-bit version of the `HttpPlatformHandler <http://www.iis.net/downloads/microsoft/httpplatformhandler>`_ on a regular (not Nano) machine. After installing you will need to copy the following files:
+
+* *%windir%\\System32\\inetsrv\\httpPlatformHandler.dll*
+* *%windir%\\System32\\inetsrv\\config\\schema\\httpplatform_schema.xml*
+
+On the Nano machine you’ll need to copy those two files to their respective locations.
+
+.. code:: ps1
+
+  Copy-Item .\httpPlatformHandler.dll c:\Windows\System32\inetsrv
+  Copy-Item .\httpplatform_schema.xml c:\Windows\System32\inetsrv\config\schema
+
+Enabling the HTTPPlatformHandler
+--------------------------------
+
+You can execute the following PowerShell script in a remote PowerShell session to enable the HTTPPlatformHandler module on the Nano server.
+
+.. note:: This script runs on a clean system, but is not meant to be idempotent. If you run this multiple times it will add multiple entries. If you end up in a bad state, you can find backups of the *applicationHost.config* file at *%systemdrive%\inetpub\history*.
+
+.. literalinclude:: nano-server/enable-platformhandler.ps1
+  :language: ps1
+
+Manually Editing *applicationHost.config*
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You can skip this section if you already ran the PowerShell script above. Though is not recommended, you can also enable the HTTPPlatformHandler by manually editing the *applicationHost.config* file.
+
+Open up *C:\\Windows\\System32\\inetsrv\\config\\applicationHost.config*
+
+Under ``<configSections>`` add
+
+.. literalinclude:: nano-server/applicationHost.config
+  :language: xml
+  :lines: 42-43
+  :emphasize-lines: 2
+
+In the ``system.webServer`` section, unlock handlers
```

Comment:
> In the ``system.webServer`` section, unlock handlers 

In the system.webServer section group, update the handlers section to allow the configured handlers to be overridden.